test/lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -193,13 +193,13 @@ test("integration: Analysis of an empty xml view", async (t) => {
 		};
```

Comment:
This looks suspicious to me. I would a) expect that JSON.stringify does the wrong thing for XML content (e.g. add surrounding quotes, escape quotes in the content) and b) that due to a) the error message should be different, not the "Invalid empty XML" message.

What am I missing?

Reply:
Done.

I also wondered about that but just copied the existing tests without thinking about it. The mockPool doesn't seem to be used to read the XML content. It just needs a findResource function that resolves.